pep8.py

```diff
@@ -2061,7 +2061,8 @@ def read_config(options, args, arglist, parser):
         # Second, parse the configuration
         for opt in config.options(pep8_section):
             if opt.replace('_', '-') not in parser.config_options:
-                print("  unknown option '%s' ignored" % opt)
+                if options.verbose:
+                    print("  unknown option '%s' ignored" % opt)
```

Comment:
@wfleming Yeah, I wanted to do that instead, but that seemed a deeper change and I was concerned about 2/3 compatibility so I punted for now. Since the surrounding code checks `verbose` for informational messages I think this is sufficient for now.